examples/enclosures.py

```diff
@@ -135,32 +141,49 @@
 bay_number = 1  # Required for C7000 enclosure
 csr_data = {
     "type": "CertificateDtoV2",
-    "organization": "",
-    "organizationalUnit": "",
-    "locality": "",
-    "state": "",
-    "country": "",
-    "commonName": ""
+    "organization": "organization",
+    "organizationalUnit": "organization unit",
+    "locality": "locality",
+    "state": "state",
+    "country": "country",
+    "commonName": "name"
 }
 try:
-    oneview_client.enclosures.generate_csr(csr_data, enclosure_uri, bay_number=bay_number)
+    enclosure.generate_csr(csr_data, bay_number=bay_number)
     print("Generated CSR for the enclosure.")
 except HPOneViewException as e:
     print(e.msg)
 
 # Get the certificate Signing Request (CSR) that was generated by previous POST.
 try:
-    csr = oneview_client.enclosures.get_csr(enclosure_uri, bay_number=bay_number)
+    csr = enclosure.get_csr(bay_number=bay_number)
     with open('enclosure.csr', 'w') as csr_file:
-            csr_file.write(csr["base64Data"])
+        csr_file.write(csr["base64Data"])
     print("Saved CSR(generated by previous POST) to 'enclosure.csr' file")
 except HPOneViewException as e:
     print(e.msg)
 
+# Import CA signed certificate to the enclosure.
 try:
-    # Get Enclosure by scope_uris
-    if oneview_client.api_version >= 600:
-        enclosures_by_scope_uris = oneview_client.enclosures.get_all(scope_uris="\"'/rest/scopes/3bb0c754-fd38-45af-be8a-4d4419de06e9'\"")
+    # Certificate has to be signed by CA before running the task.
+    certificate_file = "enclosure.csr"
+    with open(certificate_file, "r") as file_object:
+        certificate = file_object.read()
+
+    certificate_data = {
+        "type": "CertificateDataV2",
+        "base64Data": certificate
+    }
+
+    enclosure.import_certificate(certificate_data, bay_number=bay_number)
+    print("Imported Signed Certificate  to the enclosure.")
+except HPOneViewException as e:
+    print(e.msg)
+
+# Get Enclosure by scope_uris
+if oneview_client.api_version >= 600:
+    try:
+        enclosures_by_scope_uris = enclosure.get_all(scope_uris="\"'/rest/scopes/a070577f-0dfa-4b86-ba48-863f3cac291e'\"")
```

Comment:
we should replace this with helper method by giving scope name and returning enclosures having that scope instead of passing scope_uris.

Reply:
You are right but we don't have a plan to add helper methods in this version. This version is for restructuring the base class to support the helper methods and upgrade the SDK to support API version 800.
A new version will be released with all the helper methods.